extension.md

```diff
@@ -0,0 +1,14 @@
+# Open Source Development
```

Comment:
move to `projects/javascript/prime-video-translate.md` or whatever you'd like to name the library